### ISSUE TITLE
Contact us: The subject for contact us is hosed, says "Dataverse ???header.contact???" but is seemingly intermittent.

### ISSUE THREAD

**Issue by kcondon** [IQSS/dataverse#2201]
Sometimes the contact us/ support link form shows ???header.contact??? in the subject, sometimes it displays nothing for subject, sometimes it displays the text entry field. 

I myself saw each within the same browser session. 

I have also received at least one such email from a user.

**Comment by mheppler** [comment]
Related to #2078.

**Comment by mheppler** [comment]
Removed subject from Support commandLink setPropertyActionListener in dataverse_header so that the contact form opens with a blank input field.

In a previous commit 7afddf84196024e4dbd3c728c08de9b9be9e4e45 from @raprasad, for #2078, he removed the conditionally rendered static text that used to read, "Dataverse Contact" from contactFormFragment.xhtml.

That alone should fix the bug here in this issue. I did find some additional code in dataverse_header.xhtml that was no longer needed, if we weren't displaying automatically generated email subject, so I removed that as well.

What I wasn't able to determine is why or even how, the subject displayed as either: A.) ???header.contact??? B.) nothing, or C.) text input field. There was dynamic code on contactFormFragment that Raman has removed for 4.0.1. That code was used for the old Feedback link that was in the header. My only assumption is that because we changed how the new Support link compared to the old Feedback link pre-populated email form with user info and subject, the form had errors, and those errors were sometimes handled differently, depending if you were logged in or not.
